### DISCUSSION TITLE
REI won't show Winged mod recipes

### DISCUSSION THREAD

**Discussion by AryanShetty** [shedaniel/RoughlyEnoughItems#894] [Q & A]
https://user-images.githubusercontent.com/16573088/172415979-b6325f35-44cd-403e-b8be-c2296eb6a24e.mp4

Hey! I just got the winged mod, and REI won't show recipes for the winged mod. What files would i need to share to help you help me? Also Idk how, but i bugged up the vanilla recipes tab, and now it wont show me any recipes, can you help me fix it? In the video attached below, I'm pressing the show recipes button continuosly, and you can see that it works for potions, but it won't work for the Winged mod items. This is a new issue, since i remember the recipes button working with the winged mod a while ago. 

After more testing, it seems that REI is not showing recipes for loads of items, vanilla included. I have shown it not working in the other attached video. The uses button works, but the recipes button is doing absolutely nothing

https://user-images.githubusercontent.com/16573088/172415512-6bf04813-af6b-45d0-9d85-1253df8b3b13.mp4

**Comment by shedaniel** [comment]
Hello, please attach your latest.log from the .minecraft/logs folder.

**Comment by AryanShetty** [comment]
[2022-06-09-4.log.gz](https://github.com/shedaniel/RoughlyEnoughItems/files/8878434/2022-06-09-4.log.gz)

Here you go!

**Reply by shedaniel** [reply]
Sorry, cracked minecraft isn't supported!
